### ISSUE TITLE
Invalid Markdown syntax on FRD.md breaks numbering

### ISSUE THREAD

**Issue by mpyne-navy** [mpyne-navy/ade-example#2]
(this is an example issue)

The Markdown syntax on FRD.md in the current `master` branch of development is invalid, leading to improper organization and numbering of the ADE requirements.

Expected behavior:

>    1.  ADE should be:
>        1. super fast
>        2. "Web scale"

etc.

Current behavior:

>    1.  ADE should be:
>    2. super fast
>    3. "Web scale"

This is a problem because the fact that "super fast" etc. doesn't nest under "ADE should be" could make understanding our requirements more confusing.

**Comment by mpyne-navy** [comment]
Once an issue is opened, it can be debated, reviewed, discussed, etc. inline in the comments here in the web interface.  Fixing it can be prioritized based on milestones, assigned to specific projects, etc., allowing for easy management and prioritization of tasks.

Note that fixes to the underlying source code are usually handled instead by using a "Pull Request", which can reference Issues and includes specific facilities for review of code changes.  These code changes can themselves occur over multiple commits to make review easier to conduct.  In fact, they usually *should* happen over multiple commits.

**Comment by mpyne-navy** [comment]
So a code fix has been merged in by now to fix this.  But the issue remains open (at least, by default) until the merged in feature can be tested and verified to fix the issue.  This can be configured as part of the CI/CD pipeline (e.g. you might have the merged commit launch an automatic testsuite that, if all tests inserted against the reported issue pass, automatically closes the issue).

Since I've verified it separately, I'll close the issue now as part of this comment.  It can be reopened if needed.
